src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -469,5 +474,17 @@ private async Task<ReviewRevisionModel> FindMatchingApprovedRevision(ReviewModel
             }
             return null;
         }
+
+        public async void UpdateReviewBackground()
+        {
+            // Enabling this only for manual reviews in the beginning to check impact on system performance
+            // We will enable it for all reviews based on the perf details
+            // Automatic reviews are already updated as part of scheduled upload daily
+            var reviews = await _reviewsRepository.GetReviewsAsync(false, "All", false);
+            foreach(var review in reviews.Where(r => IsUpdateAvailable(r)))
```

Comment:
I'm a bit worried this tight loop would kill the service...

Reply:
I am not sure if C# has any built in option to queue and delay the run . One simple approach is to add a sleep between runs to avoid taking all of processing power by this task. Initial run is going to take longer but further runs should reduce scope.

Reply:
I have added delay of 5 sec between each reviews. Please let me know if you have any better solution.